README.md

```diff
@@ -13,44 +13,52 @@ Other than tracing, Tracee is also capable of capturing files written to disk or
 
 [Check out this quick demo of tracee](https://youtu.be/WTqE2ae257o)
 
-## Getting started
+## Getting Started
 
 ### Prerequisites
 
-To run, Tracee requires the following:
-- Linux kernel version > 4.14
-- Kernel headers
+To run, Tracee requires Linux kernel version > 4.14, and the relevant kernel headers available under `/usr/src`.
+The rest of dependencies are already built into the Docker image:
 - C standard library (currently tested with glibc)
-- clang > 9
+- libelf, zlib libraries
```

Comment:
This is not needed to run tracee.
Only to compile it

Reply:
are you sure? (not suggesting otherwise, just asking)

Reply:
libbpf.a is statically compiled (.a), and it uses libelf during it's compilation, so I'm quite sure, but worth testing this

Reply:
I'm not sure but isn't it needed to read the obj file?

Reply:
I think that when we linked tracee with libbpf.a, the needed code which reads the obj file is already part of the code

Reply:
```
error while loading shared libraries: libelf.so.1: cannot open shared object file: No such file or directory
```

Reply:
Ok, good to know. Indeed was worth the test :-)